### ISSUE TITLE
Evaluation of defended Model

### ISSUE THREAD

**Issue by asthaverma13** [CEA-LIST/adv-reid#4]
After training the defended model GOAT, when i am attacking this model, it is giving accuracy as
rank1 : mean = 0.0008907363517209888, max = 0.0008907363517209888, min = 0.0008907363517209888, std = 0.0
rank5 : mean = 0.0029691210947930813, max = 0.0029691210947930813, min = 0.0029691210947930813, std = 0.0
rank10 : mean = 0.00801662728190422, max = 0.00801662728190422, min = 0.00801662728190422, std = 0.0
map : mean = 0.0020973790522167803, max = 0.0020973790522167803, min = 0.0020973790522167803, std = 0.0
Total computing time : 161.57796907424927 sec

Whereas, as stated in the paper, the accuracy should be higher after attacking the defended model.

Please provide me the checkpoints of defended model.

Thanks and Regards
Astha Verma
PhD Scholar

**Comment by qbouniot** [comment]
Hello !
Thank you for your interest in our work. 
Can you tell me which command did you run to train the model ? I'm currently reviewing the code to correct some mistakes. 
I will upload the checkpoints soon, they should be available in the following week.

**Comment by asthaverma13** [comment]
Thanks for your reply.
I used the following command for defending the model by giving my arguments in place of output_folder, dataset_folder, dataset and checkpoint_name

python train.py \
        <output_folder> \
        <dataset_folder> \
        --dataset <dataset> \
        --lr 0.00035 \
        -wd 5e-4 \
        -n 60 \
        -f <checkpoint_name> \
        -b 32 \
        --classif \
        --pretrained \
        --adv \
        --inter \
        --pull

Please upload the defended model as soon as possible. 

Astha

**Comment by qbouniot** [comment]
Hello ! We have uploaded checkpoints of the defended models, the links are now in the readme. I hope this will help you.

**Comment by asthaverma13** [comment]
Thanks a lot. I will check it out.

Astha
